### ISSUE TITLE
MA: Update people data

### ISSUE THREAD

**Issue by bhrutledge** [openstates/issues#211]
Following up on https://github.com/openstates/issues/issues/207. Blocked by https://github.com/openstates/people/pull/374.

Based on my read of the [contributing guide](https://docs.openstates.org/en/latest/contributing/people.html) and some experimentation, I think the process will be something like:

- Update MA `people` scraper output
    - Probably just `Capitol office -> Capitol Office`
- Run MA `people` scraper
- Run `to_yaml.py`
    - This should handle the `email` schema change described in <https://github.com/openstates/issues/issues/207#issuecomment-752134795>
- Run `merge.py --incoming`
    - Fix removal of `start_date`
    - Add special case for `contact_details` to avoid appending duplicate entries
    - `git rm` unmodified YAML files with outdated rep names (e.g. [`Bradford-R-Hill`](https://github.com/openstates/people/blob/main/data/ma/legislature/Bradford-R-Hill-6e64e8db-c1f5-4f72-994b-dbe9cfdab8a5.yml) will be `Bradford-Hill`)
- Handle new and retired people
    - There's `vacancies` in [openstates/metadata/_creation/settings.yml](https://github.com/openstates/openstates-core/blob/c5e33f2e9d5e06c8ddf94bd51fdea98bb85d4548/openstates/metadata/_creation/settings.yml#L316-L331) and [people/settings.yml](https://github.com/openstates/people/blob/471905141ed1aab580303cc3bedd7abe3f819692/settings.yml#L22-L35), and they're slightly different. Does one take precedence?

Depending on the timing, maybe it's worth doing this in two batches: updating the current data (since some vacancies have been filled), and then updating for the new legislative session, once https://malegislature.gov/Legislators/ has been updated.

Outdated `vacancies`, with comments for current legislators:

```yaml
- chamber: upper
  district: Plymouth and Barnstable
  vacant_until: 2020-05-01
  # https://malegislature.gov/Legislators/Profile/SLM0
- chamber: upper
  district: Second Hampden and Hampshire
  vacant_until: 2020-05-01
  # https://malegislature.gov/Legislators/Profile/JCV0
- chamber: lower
  district: Third Bristol
  vacant_until: 2020-05-01
  # https://malegislature.gov/Legislators/Profile/CAD1
- chamber: lower
  district: Thirty-Second Middlesex
  vacant_until: 2020-05-01
  # https://malegislature.gov/Legislators/Profile/KLG1
- chamber: lower
  district: Thirty-Seventh Middlesex
  vacant_until: 2020-05-01
  # https://malegislature.gov/Legislators/Profile/DAS1
```

**Comment by bhrutledge** [comment]
Re: multiple `vacancies`, from @jamesturk in https://github.com/openstates/openstates-core/pull/18#issuecomment-752683555:

> (the ones in openstates-core) are ignored, a relic of another version I didn't realize still existed. I'll clean those up.

**Comment by bhrutledge** [comment]
@jamesturk With the merge of https://github.com/openstates/people/pull/401, can this be closed?

**Comment by jamesturk** [comment]
yes 👍

**Comment by bhrutledge** [comment]
Awesome. If/when you have the time and inclination, I'm curious about your process for #401, and how closely it followed what I outlined in https://github.com/openstates/issues/issues/211#issue-776492455.

**Comment by jamesturk** [comment]
Yep, your process was just about right from what I recall (been a bit of a busy 2 weeks updating all of these, but I don't remember anything else standing out for MA)

- Fix removal of start_date
- Add special case for contact_details to avoid appending duplicate entries

These two were mostly handled already by other updates for this year, so not a ton to do on them.

- For vacancies, people/settings.yml is the right place (the other is now gone) and I removed all vacancies since this was a new term.  

The only weird piece was that to avoid having to run the script that creates districts I ran a quick script against the production database to update the district names in place before doing the import into production.  This made things simpler and I ran it locally on my dev database to verify, then ran in production right before the import of people happened.
